src/components/spaces/populations/populations-charts.tsx

```diff
@@ -0,0 +1,163 @@
+    });
+  }
+
+  public componentWillUnmount() {
+    if (this.disposer) this.disposer();
+  }
+
+  public render() {
+    const { chartData, isPlaying } = this.props;
+    const model = this.stores.populations.model as MousePopulationsModelType;
+    const { showMaxPoints, toggleShowMaxPoints, enablePieChart, showPieChart, toggleShowPieChart } = model;
+    const toggleButtonTitle = showMaxPoints ? "Show Recent Data" : "Show All Data";
+    const topChartClassName = "top-chart" + (showPieChart ? "" : " hidden");
+    const bottomChartClassName = "bottom-chart" + (showPieChart ? " small" : "");
+    return (
+      <div className="chart-container">
+        <div className="chart-header">
+          <div>
+            { enablePieChart &&
+            <div className={"button-holder small-icon" + (showPieChart ? " active" : "")}>
+              <svg className="icon" onClick={toggleShowPieChart}>
+                <use xlinkHref="#icon-pie-chart" />
+              </svg>
+            </div>
+            }
+          </div>
+          <div className="title">
+            { chartData.name }
+          </div>
+          <div>
+            <div className="button-holder" onClick={toggleShowMaxPoints}>
+              { toggleButtonTitle }
+            </div>
+          </div>
+        </div>
+        <div className="charts">
+          { enablePieChart &&
+          <div className={topChartClassName}>
+            { this.renderPieChart(true) }
+            { this.renderPieChart(false) }
+          </div>
+          }
+          <div className={bottomChartClassName}>
+            <Chart
+              chartData={chartData}
+              chartType={"line"}
+              isPlaying={isPlaying}
+              hideTitle={true}
+              height={this.state.lineChartHeight}
+            />
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  private renderPieChart = (initialData: boolean) => {
+    if (!this.props.chartData.dataSets.length || !this.props.chartData.dataSets[0].dataPoints.length) {
+      return (<div />);
+    }
+
+    let pointIdx = 0;
+    let date = 0;
+    if (!initialData && this.props.chartData.dataSets[0] && this.props.chartData.dataSets[0].dataPoints.length > 0) {
+      pointIdx = this.props.chartData.dataSets[0].dataPoints.length - 1;
+      date = Math.floor(this.props.chartData.dataSets[0].dataPoints[pointIdx].a1);
+    }
+    const text = initialData ? "Initial: 0 months" : `Current: ${date} months`;
+    // create single array of points, either the first of each dataset or the last
+    let data = this.props.chartData.dataSets.filter(ds => ds.display).map(ds => {
+        const point = ds.dataPoints[(initialData || !ds.dataPoints.length) ? 0 : ds.dataPoints.length - 1];
+        const value = point && !isNaN(point.a2) ? point.a2 : 0;
+        return {
+          label: ds.name,
+          value,
+          color: ds.color as string
+        };
+    }) as PieChartData[];
+    data = data.filter(d => d.value);
```

Comment:
I wonder if the pie chart data needs to specify how many decimal digits to use when displaying the percents on the pie chart?  At present the pie chart is rounding to an integer (which was the original request), but the addition of this population level pie chart and the addition of the heredity level mutations means that we might need to display decimal digits (at least so we don't round values down to 0 which will confuse the user).  We show 2 decimal digits in the line chart hover and Frieda requested 1 decimal digit in the heredity level pie chart after looking at the mutation branch.  It doesn't need to be tackled now, but probably needs to be addressed during this pie chart work.

Reply:
Agree with further discussion for later. The specs all show whole numbers, and this actually makes sense to me, but could understand thinking about using decimals. (Note, we shouldn't ever get 1% let alone >0.5% rounding down to 0, since the max carrying capacity of the environment is 50.)

Reply:
Right.  And I should have been clear that I was conflating some of the issues that were seen in the heredity level pie charts when mutations were turned on.  It seems like this whole issue should probably be a separate story since it's tied to the work here in the population level and the mutation work in the heredity level.  The story should probably answer the following:
How do we want the pie chart numbers formatted?  
How do we want to handle visibility and display of the pie chart numbers?

Reply:
One thing (again, future discussion) that might help would be roll-over (plus click, four touch devices) of slices, which could show the exact value in the pop-up that appears (and maybe also raw numbers, not just percent). That would also potentially alleviate the crowded-labels issue. (I am not looking forward to thinking about dynamically generating external labels for small slices.)

Reply:
Yup, I groaned when I imagined creating and positioning the labels on the fly.  Maybe there's a more obvious solution?  Lots of folks must have this issue.